YSI_Data/y_foreach/features.md

```diff
@@ -152,7 +152,7 @@ case, we need to return `-1` when we run out of positive even integers. The
 last number available in signed 32bit integers is `2147483647` - defined in PAWN
 as `cellmax`, but this is odd so the last positive integer must be 1 less than
 that, i.e. `2147483646`, which can be written out in full or calculated as
-`cellmax - 1`. Therefore, when the input to the special iterator is the last
+`cellmax - 1`. Therefore, when the input to the special iterator is the least

```

Comment:
That totally reverses the meaning of the sentence from the highest positive value to the lowest.